test/Table_Tests/src/Util.enso

```diff
@@ -50,15 +50,17 @@ type Test_Failure_Error
     to_display_text self = "Test failure error: "+self.message
 
 ## PRIVATE
-table_should_equal_impl actual expected loc =
-    case expected of
+table_should_equal_impl actual_table expected_table loc ignore_order =
+    case expected_table of
         _ : Table ->
+            actual = if ignore_order then actual_table.sort else actual_table
+            expected = if ignore_order then expected_table.sort else expected_table
             if actual.columns.length != expected.columns.length then
                 Panic.throw (Test_Failure_Error.Error 'Tables differ at '+loc+'.\nActual:\n'+actual.display+'\nExpected:\n'+expected.display+'\nExpected '+expected.columns.length.to_text+" columns, but got "+actual.columns.length.to_text+'.')
             Panic.catch Test_Failure_Error (actual.columns.zip expected.columns a-> e->(column_should_equal_impl a e)) error->
                 msg = 'Tables differ at '+loc+'.\nActual:\n'+actual.display+'\nExpected:\n'+expected.display+'\n'+error.payload.message
```

Comment:
Please note that after these changes this no longer prints the original tables that were tested but the _sorted_ tables.

IMO this may be confusing as when debugging I'd probably be surprised why I see the table sorted when in the code (excluding this test code) I see no sorting. If I remember there is `sort` in `should_equal` - ok, but it's not obvious.

I guess we can fix that if we add a suffix, like: "Actual (sorted):" and "Expected (sorted):" if the `ignore_order` is set.

Reply:
Done